### ISSUE TITLE
Finder -> iTerm command launches session named "Default" instead of using the one marked as default

### ISSUE THREAD

**Issue by richardtessier** [LeEnno/alfred-terminalfinder#12]
The **fi** command in **TerminalFinder.alfredworkflow** should use "" instead of "Default" to launch a session with the **default** Profile instead of one with the Profile named **Default**.

```applescript
tell myTerm
    if not isRunning and termCount is not 0 then
        set mySession to current session
    else
        set mySession to (launch session "Default") -- replace "Default" with "" on this line
    end if
    tell mySession to write text "cd " & pathList
end tell
```

The **src/fi.scpt.txt** file has the correct line already so I guess an export might be all that's needed.

**Comment by LeEnno** [comment]
Thanks for reporting. It's exactly what you said, #7 fixed this in the script file, but not the actual workflow. Will incorporate that and a few other things within the next days..

**Comment by richardtessier** [comment]
Thanks, great workflow.

**Comment by LeEnno** [comment]
Please redownload the workflow. Should be working now.
I merged #11 so it's now `"Default Session"` instead of just `""` but it will should do the trick.
